lib/dockerregistry/inventory.go

```diff
@@ -102,13 +102,13 @@ func (sc *SyncContext) Fatal(v ...interface{}) {
 }
 
 // ParseManifestFromFile parses a Manifest from a filepath.
-func ParseManifestFromFile(filePath string) (Manifest, error) {
+func ParseManifestFromFile(filePath string, allowEmptyImages bool) (Manifest, error) {
```

Comment:
The allowEmptyImages is no longer required as it is optionally allowed since 6cdac03ad6eff62f63ab3da91b9913ea95edda22

Reply:
Done :-)